tests/test_cognitoidp/test_cognitoidp.py

```diff
@@ -137,6 +137,7 @@ def test_create_user_pool_domain():
     user_pool_id = conn.create_user_pool(PoolName=str(uuid.uuid4()))["UserPool"]["Id"]
     result = conn.create_user_pool_domain(UserPoolId=user_pool_id, Domain=domain)
     result["ResponseMetadata"]["HTTPStatusCode"].should.equal(200)
+    result["CloudFrontDomain"].should_not.be.none
```

Comment:
Typo - this should be `result["DomainDescription"]["CloudFrontDomain"]`

Reply:
Kind of confused... that structure would not match the next test result
`result["CloudFrontDomain"].should.equal("e2c343b3293ee505.cloudfront.net")`

The test log is gigantic and takes forever to load on github. Will try to take a look again when I get to the office.

Reply:
Fixed